c_binding/include/libstoragemgmt/libstoragemgmt_types.h

```diff
@@ -340,6 +372,56 @@ typedef enum {
 #define LSM_SYSTEM_CACHE_PCT_NO_SUPPORT             -2
 #define LSM_SYSTEM_CACHE_PCT_UNKNOWN                -1
 
+#define LSM_VOLUME_WRITE_CACHE_POLICY_UNKNOWN       0
```

Comment:
We're using -1 for unknown in some cases, and 0 here. Should we pick a consistent value for *_UNKNOWN macros? At least for new ones, anyway?